vendor/README.rst

```diff
@@ -12,7 +12,7 @@ newer release, add ``--remote``)::
 Then build ``llhttp``::
 
     cd vendor/llhttp/
-    npm install
+    npm ci
```

Comment:
What is this change? I'm not familiar with this command, and the llhttp instructions appear to still tell users to run `npm install`.

Reply:
Oh right, this is the clean install. I'll test it out shortly, as there is a new llhttp to upgrade to.

Reply:
> ...and the llhttp instructions appear to still tell users to run `npm install`.

What can I say, they should fix it too, along with a few other node-related irregularities. They correctly use it in the Dockerfile, but not in their workflows.  I'll give them a PR. 😉 

If you're developing and okay with mutating the lock file, `npm install` and others are fine.  But for CI, releases, use as a submodule, etc, `npm ci` should be used to treat it as immutable.  That's why it exists.

Reply:
https://github.com/nodejs/llhttp/pull/298

Reply:
I tried this command for upgrading to 9.2 and everything seems to work fine, so I'm happy with this.